clusters/app.ci/multi-arch/namespaces.yaml

```diff
@@ -4,11 +4,11 @@ items:
 - apiVersion: v1
   kind: Namespace
   metadata:
-    name: coreos-arm64
+    name: azure-arm64
```

Comment:
is it required?

Reply:
I got this error on `multi-arch-gen-valid` test
```
ERROR: To update the the multi-arch image mirroring run the following and
ERROR: commit the result:
ERROR: $ make multi-arch-gen
```

Reply:
Apparently it returns a different order in my env. I think ``ruamel.yaml`` has some regression which makes it returning stuff in different order sometimes
I recreated this change in https://github.com/openshift/release/pull/34728 without reordering